<launch>
    <!-- DESCRIPTION -->
    <!-- This launch files is use to control turtle runing in pentagon shape. 
    It implemnts trajectary parameters from ymal file, and plot path error in 
    rqt graph  -->
    <node pkg="tsim" type="turtle_way" name="turtle_way"  output="screen">
    </node>

    <!-- <node pkg="tsim" type="reset_sever" name="reset_sever"  output="screen"> -->
    <!-- </node> -->

    <node name="turtlesim_node" pkg="turtlesim" type="turtlesim_node" />
    <node name="pos_vel_plot" pkg="rqt_plot" type="rqt_plot" args="/pose_error" />

    <!-- <param name="x" value="3" />
    <param name="y" value="2" />
    <param name="width" value="4" />
    <param name="height" value="5" />
    <param name="trans_vel" value="2" />
    <param name="rot_vel" value="1" />
    <param name="frequency" value="100" /> -->
    <rosparam command="load" file="$(find tsim)/config/turtle_way.yaml" />

</launch>
